### DISCUSSION TITLE
I need more help

### DISCUSSION THREAD

**Discussion by FallenCB** [KadeArchive/Kade-Engine#2236] [Q&A]
whenever i try to add a character and i select him in the debug menu, it says Null Object Reference

**Comment by NK-523** [comment]
What that means is its trying to call on something that doesn't exist, check you characters.hx code and compare the sprite names with the names in your code.

**Reply by NK-523** [reply]
I had a similar issue and I ended up just misspelling my character's name in the .xml

**Reply by FallenCB** [reply]
It does the same thing, any other suggetions?

**Reply by NK-523** [reply]
make sure your character.hx code is setup kind of like this

case 'dad':
tex = Paths.getSparrowAtlas('DADDY_DEAREST','shared',true);
frames = tex;
animation.addByPrefix('idle', 'Dad idle dance', 24, false);
animation.addByPrefix('singUP', 'Dad Sing Note UP', 24, false);
animation.addByPrefix('singRIGHT', 'Dad Sing Note RIGHT', 24, false);
animation.addByPrefix('singDOWN', 'Dad Sing Note DOWN', 24, false);
animation.addByPrefix('singLEFT', 'Dad Sing Note LEFT', 24, false);

loadOffsetFile(curCharacter);
